client/state/action-types.js

```diff
@@ -65,6 +65,11 @@ export const COUNTRY_STATES_REQUEST_SUCCESS = 'COUNTRY_STATES_REQUEST_SUCCESS';
 export const CURRENT_USER_FLAGS_RECEIVE = 'CURRENT_USER_FLAGS_RECEIVE';
 export const CURRENT_USER_ID_SET = 'CURRENT_USER_ID_SET';
 export const DESERIALIZE = 'DESERIALIZE';
+export const DIRECTLY_ASK_QUESTION = 'DIRECTLY_ASK_QUESTION';
+export const DIRECTLY_INITIALIZE = 'DIRECTLY_INITIALIZE';
+export const DIRECTLY_MAXIMIZE = 'DIRECTLY_MAXIMIZE';
+export const DIRECTLY_MINIMIZE = 'DIRECTLY_MINIMIZE';
```

Comment:
Right — we have no way to hook into widget-initiated events, so these actions are not being tracked in state.

Currently there's no plan for `MAXIMIZE`, `MINIMIZE`, or `OPEN_ASK_FORM`. (I could see us using min/max as we go through some UX testing though). I figured I'd include them because this is actually the entire Directly API, so it saves future devs from having to dive into Directly's docs. But if it's better to simply eliminate them for now, I'm amenable to that.

Might it also be best to remove them from [the `lib/directly` API](https://github.com/Automattic/wp-calypso/blob/f539d81a9c93bb9007b7feb4c673248ca5af684e/client/lib/directly/index.js#L31-L41)?